tests/test_modeling.py

```
@@ -13,370 +13,325 @@
+
         # words_mask[1, 2] = 1 means token at position 2 goes to word position 0
-        assert torch.allclose(
-            words_embedding[1, 0],
-            basic_setup['token_embeds'][1, 2]
-        )
-    
+        assert torch.allclose(words_embedding[1, 0], basic_setup["token_embeds"][1, 2])
+
     def test_creates_valid_mask(self, basic_setup):
         """Should create mask based on text_lengths."""
         words_embedding, mask = extract_word_embeddings(**basic_setup)
```

Ruff (0.15.1)

[warning] Unpacked variable `words_embedding` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)